scout/server/blueprints/variants/controllers.py

```diff
@@ -1108,7 +1182,35 @@ def variant_export_lines(store, case_obj, variants_query):
         else:
             variant_line.append("N/A")
 
-        if case_obj.get("track") == "cancer":
+        if category == "fusion":
```

Comment:
This code adds quite a lot of complexity to an already complex function and it would be much better to move outside.

Complexity of main branch with radon:

<img width="302" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/cfa92d31-2be4-4e78-bd9e-f6791b2d72b1">


Complexity of this branch:

<img width="318" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/ee6bed2e-3861-43a3-8f73-1d8b8f18c138">

Reply:
Right, I would still argue it is not complexity, rather something like affine crystallinity or redundancy, but I do agree it should be fixed.